crates/turbopack-css/src/util.rs

```diff
@@ -0,0 +1,50 @@
+use lazy_static::lazy_static;
+use regex::Regex;
+
+pub fn stringify_str(str: &str) -> String {
+    lazy_static! {
+        static ref CSS_STR_ESCAPE_CHARS: Regex =
+        // Escape the following characters required for strings by the CSS scanner:
+        // https://www.w3.org/TR/CSS21/grammar.html#scanner
+        // https://github.com/vercel/turbo/pull/2598#discussion_r1022625909
+        //
+        // Note that the form feed character is not representable as \f in Rust strings, so
+        // the unicode representation \u{0c} is used.
+        regex::Regex::new("(\"|\n|\r|\\\\|\u{0c})").unwrap();
```

Comment:
Should this be searching for a single backslash instead of a double?

Reply:
(discussed async) this needs to be doubled since it's a regular expression inside of a Rust string literal (must be escaped once for the string literal, then to appear as a literal backslash in the regex).